examples/training/object_detection/pascal_voc/retina_net.py

```diff
@@ -127,6 +124,24 @@ def unpackage_tfds_inputs(inputs, bounding_box_format):
         ),
     ]
 )
+
```

Comment:
Not crucial, but it seems like you could easily package all these transforms into one augmenter instead of doing many maps

Reply:
you would hope so, but that is not the case.  RandAugment doesn't work on dictionary inputs at this point.